include/operations/blas3_trees.h

```diff
@@ -306,18 +306,19 @@ make_gemm(input_t buffer_a, input_t buffer_b, output_t buffer_c,
  * @Note This kernel assumes the column-major matrices
  * @Note This kernel uses fixed size blocks of 16, but this can be changed
  */
-template <bool UnitDiag, bool Upper, int BlockSize, typename matrix_t>
+template <bool UnitDiag, bool Upper, int BlockSize, typename lhs_t,
```

Comment:
I don,t think this changes related to const qualifier

Reply:
This name change was required to differentiate between the input and output container types. The input in this case can be const and the output cannot be const, thus we needed separate template parameters to represent the two different types.